internal/api/genapi/input.go

```diff
@@ -167,6 +167,18 @@ var inputStructs = []*structInfo{
 		outFile:     "scopes/key_version.gen.go",
 		skipOptions: true,
 	},
+	{
+		inProto:     &scopes.KeyVersionDestructionJob{},
+		outFile:     "scopes/key_version_destruction_job.gen.go",
+		skipOptions: true,
+		fieldOverrides: []fieldInfo{
+			// int64 fields get marshalled by protobuf as strings, so we have
+			// to tell the json parser that their json representation is a
+			// string but they go into Go int64 types.
+			{Name: "CompletedCount", JsonTags: []string{"string"}},
+			{Name: "TotalCount", JsonTags: []string{"string"}},
+		},
```

Comment:
Note that this is taking advantage of a feature of the CLI generation that will likely change soon - it was merged for 0.11.0 to get support for stringified large integers into 0.11.0 but longer term this will likely be implicit behavior when using large integers in the protobuf type. CC @tmessi @hugoamvieira